### ISSUE TITLE
Search for builds

### ISSUE THREAD

**Issue by lightyearsaway** [dofuslab/dofuslab#347]
Throwing some ideas out there:
NLP to understand what people are looking for, like "int/agi cra" vs "nomarow cra" vs "tormenting 200"
Should also search for users and folders, maybe in different tabs in the search results like LinkedIn's search
Might also include work for implicitly tagging users' builds that they don't tag themselves, like any build that has over 800 chance could be tagged with the chance tag, or any build that is in the top 10% of builds at that level in amount of chance
Basically a huge undertaking, but could be a strong differentiator vs competitors
